backend/qemu.pm

```diff
@@ -490,6 +490,7 @@ sub start_qemu {
     bmwqemu::save_vars();                 # update variables
 
     mkpath($basedir);
+    runcmd('/usr/bin/chattr', '+C', $basedir);
```

Comment:
yeah, some module I found just hardcoded the values - which isn't cool either.

I don't have a problem with adding such a call to XS though :)